### PR TITLE
#182 Profile page last features update

### PR DESCRIPTION
**Pull request by EricKokmanian**
-Updated profile page tab
-Add activity progress bar
-Display all of user's questions
-Fix issue with link to old defective profile page

### CODE REVIEW

File: App/PHP_HTML/login sample/sign_in_submission.php [hunk 1]
```diff
@@ -33,7 +33,7 @@
             <ul class="nav navbar-nav navbar-right">
                 <li><a href="#">Dashboard</a></li>
                 <li><a href="#">Settings</a></li>
-                <li><a href="profile.php">Profile</a></li>
+                <li><a href="#">Profile</a></li>
```
Comment (refatK) [inline App/PHP_HTML/login sample/sign_in_submission.php]: Could you explain this.

File: App/PHP_HTML/profile.php [hunk 2]
```diff
@@ -84,68 +95,85 @@ function openTab(evt, cityName) {
...
             <tr>
                 <td>
-                    <ul class="">
-                        <li class=""><strong class="">Questions asked: <?php echo $user['user_questions_count']; ?></strong></li>
-                        <li class=""><strong class="">Answers: <?php echo $user['user_answers_count']; ?></strong></li>
-                        <li class=""><strong class="">Likes</strong></li>
-                        <li class=""><strong class="">Followers</strong></li>
-                    </ul>
+				<ul>
+					<div class="">
```
Comment (refatK) [inline App/PHP_HTML/profile.php]: any point to empty class name?

File: App/PHP_HTML/login sample/sign_in_submission.php (hunk 1 above)
Comment (EricKokmanian) [reply App/PHP_HTML/login sample/sign_in_submission.php]: We had an old defective profile.php file in login sample so clicking on the profile link after having signed in would direct to that page and not the correct one.

File: App/PHP_HTML/profile.php (hunk 2 above)
Comment (EricKokmanian) [reply App/PHP_HTML/profile.php]: not anymore... had plans to add a class to deal with user subs, but gave up on them :p

**Pull request merged**